oshi-core/src/test/java/oshi/software/os/OperatingSystemTest.java

```diff
@@ -100,8 +100,7 @@ public void testOperatingSystem() {
         assertTrue(proc.getStartTime() >= 0);
         assertTrue(proc.getBytesRead() >= 0);
         assertTrue(proc.getBytesWritten() >= 0);
-        assertTrue(proc.getBitness() >= 0);
-        assertTrue(proc.getBitness() <= 64);
+        assertTrue(proc.getBitness() == 64||proc.getBitness() == 32||proc.getBitness() == 0);
```

Comment:
I like this improvement.  You could also test <= against OS bitness (You can't have a 64-bit process on a 32-bit OS).

Reply:
This works:
```
        if (os.getBitness() == 32) {
            assertEquals("Process on 32-bit OS must have bitness 0 or 32", 0, proc.getBitness() & ~32);
        } else {
            assertEquals("Process on 64-bit OS must have bitness 0, 32, or 64", 0, proc.getBitness() & ~(32 + 64));
        }
```